src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -91,28 +92,30 @@ if the "same" data were presented in a different data view (as by, say, being
 transformed, cached, saved, or whatever), that the IDs between the two
 different data views would have any discernable relationship.
 
-Since this ID is practically often derived from the IDs of some other
-`ICounted` (for example, for a transform, the IDs of the output are usually derived
-from the IDs of the input), it is not only necessary to claim that the ID
-generated here is probabilistically unique, but also describe a procedure or
-set of guidelines implementors of this method should attempt to follow, in
-order to ensure that downstream components have a fair shake at producing
-unique IDs themselves.
+This ID is practically often derived from the IDs of some other `DataViewRow`.
+For example, when applying `ITransformer.Transform`, the IDs of the output are
+usually derived from the IDs of the input. It is not only necessary to claim
+that the ID generated here is probabilistically unique, but also describe a
+procedure or set of guidelines implementors of this method should attempt to
+follow, in order to ensure that downstream components have a fair shake at
+producing unique IDs themselves.
 
 Duplicate IDs being improbable is practically accomplished with a
-hashing-derived mechanism. For this we have the `UInt128` methods `Fork`,
-`Next`, and `Combine`. See their documentation for specifics, but they all
-have in common that they treat the `UInt128` as some sort of intermediate hash
-state, then return a new hash state based on hashing of a block of additional
-'bits.' (Since the bits hashed may be fixed, depending on the operation, this
-can be very efficient.) The basic assumption underlying all of that collisions
+hashing-derived mechanism. For this we have the `DataViewRowId` methods
+`Fork`, `Next`, and `Combine`. See their documentation for specifics, but they
```

Comment:
>See their documentation for specifics, [](start = 31, length = 38)

should it link?

Reply:
Well, perhaps. But I'm unsure of how to do that from code documentation to XML documentation. Since this is intended as code documentation moreso than user documentation, I would probably not want to link to, say, the docs website. Do you have a suggestion?

Reply:
So, I'm probably not going to do this, since I'm not sure how to do this 'reverse' lookup. I think I've seen situations where we are able to link .md files from the XML, IIRC, but I've never seen the reverse. Maybe using the doc stuff.